.prettierignore

```diff
@@ -1 +1,6 @@
-/data/textile/activities.json
\ No newline at end of file
+*.yml
+*.yaml
+*.html
+public/icomoon/style.css
+public/vendor/rapidoc-9.3.4.min.js
```

Comment:
Actually, you could exclude whatever is under `public/vendor/*`

Reply:
✅